utils/repository.py

```diff
@@ -7,3 +7,10 @@ def convert_list_to_tuple_string(ids_list):
         else str(tuple(ids_list))
     )
     return result
+
+
+def get_string_without_empty_spaces(string: str):
```

Comment:
This functions seems to be created only for testing the queries. Maybe we could move it to the querybuilder test file. Also consider using the following logic that does the same : `re.sub('[\s]+','', expected_query)`.  Also, it could be called `remove_white_spaces`

Reply:
This functions is called in quer_builder_test and time_entries_query_builder_test 
At the beginning, I had it in both files, but as it would be repeated code I changed it to utils
Do you think it would be better to leave it in both files instead of utils?
I'll rename the function

Reply:
Sorry. I see the logic again this `re.sub('[\s]+','', expected_query)` may not work.

Reply:
Try this
```
re.sub("[\s]+",' ', quer).strip()
```